parodos-model-api/src/main/java/com/redhat/parodos/workflow/task/parameter/WorkFlowTaskParameterType.java

```diff
@@ -27,7 +27,11 @@
  */
 public enum WorkFlowTaskParameterType {
 
-	PASSWORD, TEXT, EMAIL, DATE, NUMBER, URL;
+	PASSWORD, TEXT, EMAIL, DATE, NUMBER, URL, SELECT, MULTI_SELECT;
+
+	public boolean isSelect() {
+		return SELECT.name().equals(name()) || MULTI_SELECT.name().equals(name());
```

Comment:
I would add a test for this just in case someone delete it something in the future.